plugins/module_utils/pulp.py

```diff
@@ -717,6 +717,24 @@ def _href(self):
         )
 
 
+class PulpDebVerbatimPublication(PulpEntity):
+    _list_id = "publications_deb_verbatim_list"
+    _read_id = "publications_deb_verbatim_read"
+    _create_id = "publications_deb_verbatim_create"
+    _delete_id = "publications_deb_verbatim_delete"
+
+    _name_singular = "publication"
+    _name_plural = "publications"
+
+    @property
+    def _href(self):
+        return (
+            "deb_publication_href"
```

Comment:
That will obviously not work, as it is the same as in line 714. Either the id was different, or this api was overshadowed.

Reply:
Honestly, I don't know exactly what this part is for. Is it Pulp v2 vs Pulp v3? Or the version of openapi itself?

Reply:
Good question. It is a work around for older versions of pulp3, namely <=3.5, using an openapi schema version 2. The API got a lot more consistent when we switched to drf-spectacular (generating openapiv3) with 3.6.

see commit 955374c2d4c26efd30238b0cc113d9d7e1eeb44e

Reply:
I'm about to say, we should ditch compatibility with that dinosaur.

Reply:
Done for this class. I suggest doing it for others in a separate patch.